.github/workflows/deploy.yaml

```diff
@@ -6,6 +6,8 @@ on:
     branches:
       - main
   pull_request:
+    paths:
+      - 'deployment/**'
```

Comment:
Just checking, should it be one `*` or two `**`?

Reply:
two `**` would allow for nested paths 🤷

Reply:
I see thanks!
xref for my future self: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths